README.md

```diff
@@ -15,26 +15,24 @@ popularized by Kubernetes.  Libpod also contains the Pod Manager tool `(Podman)`
 
 ## Overview and scope
 
-At a high level, the scope of libpod and Podman is the following:
+At a high level, the scope of Podman and `libpod` is the following:
```

Comment:
You've libpod highlighted, but not Podman.  I'd vote to highlight both, but I think they should be the same.